src/common/row_set.h

```diff
@@ -185,13 +195,13 @@ class PartitionBuilder {
     for (size_t i = 0; i < blocks_offsets_.size()-1; ++i) {
       size_t n_left = 0;
       for (size_t j = blocks_offsets_[i]; j < blocks_offsets_[i+1]; ++j) {
-        mem_blocks_[j].n_offset_left = n_left;
-        n_left += mem_blocks_[j].n_left;
+        mem_blocks_[j][0].n_offset_left = n_left;
```

Comment:
Where is elements other than 0 used?

Reply:
as separate buffers were allocated for each tasks (to allow thread locality) there is no other than 0 usage.
Changed to `.data()` usage